File: guanabara-java/src/com/danillo/algoritmos/lista08/Exercicio84.java
package com.danillo.algoritmos.lista08;

import java.util.Scanner;

public class Exercicio84 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        String[] vetor1 = new String[5];
        int[] vetor2 = new int[5];

        for(int i=0; i< 5; i++){
            System.out.println("Insira o nome: ");
            vetor1[i] = entrada.nextLine();

            System.out.println("Insira a idade: ");
            vetor2[i] = entrada.nextInt();

            entrada.nextLine();
        }

        System.out.println("Pessoas menores de 18 anos: ");
        for(int i=0; i < 5; i++){
            if(vetor2[i] < 18){
                System.out.println("Nome: " +vetor1[i]+ " Idade:" +vetor2[i]);
            }
        }
    }
}

File: guanabara-python/src/lista08/exercicio84.py
vetor_a = []
vetor_b = []

for i in range(5):
    nome = input("Insira o nome: ")
    vetor_a.append(nome)
    idade = int(input("Insira a idade: "))
    vetor_b.append(idade)

for i in range(5):
    if vetor_b[i] < 18:
        print(f"Nome: {vetor_a[i]}. Idade: {vetor_b[i]} anos.")
